tests/integration/test_lists/qa/examples_test_list.txt

```diff
@@ -479,6 +479,7 @@ test_e2e.py::test_ptp_quickstart_bert[TRTLLM-BertForSequenceClassification-bert/
 test_e2e.py::test_ptp_star_attention_example[Llama3.1-8B-BF16-llama-3.1-model/Meta-Llama-3.1-8B]
 test_e2e.py::test_trtllm_bench_pytorch_backend_sanity[meta-llama/Llama-3.1-8B-llama-3.1-8b-hf-nvfp4-False-False]
 test_e2e.py::test_trtllm_bench_mgmn
+test_e2e.py::test_trtllm_bench_llmapi_launch
```

Comment:
Need to waive the TRT case since the support for TRT engine building is only available on the main branch.